helpers/deploymentConfig.ts

```diff
@@ -281,6 +310,22 @@ export const getGlobalConfig: () => Promise<NetworkConfig> = async () => {
           borrowCap: convertToUnit("55", 18),
           vTokenReceiver: "0xD5bAa0C3d61Ba3f4899565f269e5f9b186AAf14B",
         },
+        {
+          name: "Venus THE",
+          asset: "THE",
+          symbol: "vTHE",
+          rateModel: InterestRateModels.JumpRate.toString(),
+          baseRatePerYear: "0",
+          multiplierPerYear: convertToUnit("0.09", 18),
+          jumpMultiplierPerYear: convertToUnit("3.5", 18),
+          kink_: convertToUnit("0.45", 18),
+          collateralFactor: convertToUnit("0.53", 18),
+          reserveFactor: convertToUnit("0.25", 18),
+          initialSupply: convertToUnit("12286", 18),
+          supplyCap: convertToUnit("2400000", 18),
+          borrowCap: convertToUnit("1200000", 18),
+          vTokenReceiver: vTreasuryAddress,
```

Comment:
The receiver will finally be `0x1c6C2498854662FDeadbC4F14eA2f30ca305104b`, if we want to update it...

Reply:
https://github.com/VenusProtocol/venus-protocol/pull/549/commits/80060f1a8e0d52e8109fabb2466d49b5430bf7a1